### DISCUSSION TITLE
添加天翼云盘时出现 Failed init storage: ocr error

### DISCUSSION THREAD

**Discussion by canfengplaeir** [AlistGo/alist#2711] [Q&A]
windows下linux子系统下安装alist，添加天翼云盘时出现 Failed init storage: ocr error。请问如何处理？

**Comment by shengshiyuanhua** [comment]
解决方案：建议使用天翼云盘客户端，勾选上不使用OCR，即可正常使用！

**Reply by Nielson-cn** [reply]
仍然无法正常使用——会显示 need img validate code: （图形验证码）

**Reply by Jiayou-Chao** [reply]
问题解决了吗？

**Reply by Goojoe** [reply]
这是因为官方OCR API炸了


---
## 方案1:自己部署OCR
https://alist.nn.ci/zh/config/global.html#ocr-api
```
docker run -d --restart=always -p 9898:9898 xhofe/ddddocr_server:main
```
设置>全局>OCR接口
![image](https://user-images.githubusercontent.com/78355492/211178937-d468fc1b-6a4d-4d69-9648-a8fd955c9cd4.png)

## 方案2:勾选不使用OCR
然后手动在验证码栏输入验证码,缺点是每次启动都需要重新输入
